src/test/java/org/cactoos/scalar/AvgOfTest.java

```diff
@@ -178,4 +178,36 @@ public void withScalars() {
         );
     }
 
+    @Test
+    public void withDoubleMaxValue() {
```

Comment:
@allanws I'm sorry, but your PR don't fix the same problem with `Integer`, `Float` and `Long`.

Reply:
@fabriciofx , it's now fixed. Thanks for your "quick" review.

Reply:
@allanws You're welcome.